Cargo.toml

```diff
@@ -38,6 +38,7 @@ num-bigint = "0.3"
 num-integer = "0.1.44"
 num-traits = "0.2"
 arrayvec = "0.7"
+hashlink = "0.8.1"
```

Comment:
curious how/why this particular implementation was chosen.

Reply:
It was the confluence of a few superficial factors:
* It was the first search result for `LinkedHashMap` on crates.io.
* It had over 6 million downloads.
* It had a few high-profile dependents, including SQLx and DuckDB.
* It had an MIT/Apache 2.0 dual license.

![image](https://user-images.githubusercontent.com/611616/191952794-f1e0db77-6ad8-4e4d-ab37-314369812976.png)
![image](https://user-images.githubusercontent.com/611616/191952838-838777cd-f434-487b-9de7-83d81e5e4e7c.png)